lib/Backends/OpenCL/kernels.cl

```diff
@@ -755,11 +755,14 @@ __kernel void softmaxgradW(__global void *mem, cl_uint32_t origDest,
 __kernel void convolutionK(__global float *dest, __global float *src,
                            __global float *filter, __global float *bias,
                            ShapeHW filterSizes, ShapeHW strides,
-                           PaddingTLBR pads, ShapeNHWC odim, ShapeNHWC idim,
+                           PaddingTLBR pads, cl_uint32_t group, ShapeNHWC odim, ShapeNHWC idim,
```

Comment:
nit: Line is longer than 80 chars. Apparently clang-format doesn't work for this file, because it has `.cl` extension. You can temporarily rename it to `.cpp` to run clang-format again. But feel free to do this in follow up PR. There may be other format changes, not related to your PR.